.changeset/waifu-token.md

```diff
@@ -0,0 +1,6 @@
+---
+"ledger-live-desktop": patch
+"@ledgerhq/live-common": patch
```

Comment:
Please remove the `ledger-live-desktop` changeset as you only modified ledger-live-common
```suggestion
"@ledgerhq/live-common": patch
```